chromium_src/third_party/blink/renderer/modules/BUILD.gn

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+source_set("browser_tests") {
+  testonly = true
+  defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
+
+  sources = [
+    "quota/navigator_storagetest.cc",
```

Comment:
browser tests should be test_name_browsertest.cc - please create a follow-up to fix

Reply:
also this is a layering violation, you cannot have browser tests inside blink. These tests could be moved into brave, but ideally they should be converted to blink::PageTestBase

Reply:
Just moved them there at @mkarolin's suggestion for https://github.com/brave/brave-core/pull/6622#discussion_r515109474